apertium-spa.spa.dix

```diff
@@ -57965,7 +57965,7 @@
 <e lm="etiquetar">       <p><l>etiquét</l>   <r>etiquet</r></p><par n="abandón/ar__vblex"/></e>
 <e lm="europeizar">      <i>europeiz</i><par n="abandon/ar__vblex"/></e>
 <e lm="europeizar">      <p><l>europeíz</l>  <r>europeiz</r></p><par n="abandón/ar__vblex"/></e>
-<e lm="evacuar">         <i>evacu</i><par n="evacu/ar__vblex"/></e>
+<e lm="evacuar">         <i>evacu</i><par n="abandon/ar__vblex"/></e>
```

Comment:
It's missing the form with á

Reply:
Sorry, I just wanted to change the "evacu/ar__vblex" verbs in my section but I wrongly changed that. I will fix it in a moment